src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+| Hinting | Enable text hinting, which improves text readability for low-resolution displays. |
+| Antialiasing | Enable antialiasing, which improves text visibility when high-resolution text is displayed on a low-resolution screen. |
+| Scaling Factor | Enable text scaling, which increases font size for HiDPI displays while retaining text sharpness. |
+
+### Keyboard & Mouse
+
+`Keyboard & Mouse` settings extend customization of keyboard and mouse input behavior.
+
+![Keyboard Mouse Tweaks](/images/gnome-tweaks-extensions/keyboard-mouse-tweaks.png)
+
+#### Keyboard
+
+| Option | Function |
+|--------|----------|
+| Show Extended Input Sources | Display additional number of input sources found in the Settings application. |
+| Emacs Input | Apply Emacs keybindings throughout the operating system. |
+| Overview Shortcut | Choose whether left or right `SUPER` keys start the Launcher. |
+
+#### Mouse
+
+| Option | Function |
+|--------|----------|
+| Acceleration Profile | Choose how mouse movement speed adapts to user gestures. |
+| Pointer Location | Pressing the `Ctrl` key creates a visual notification at the mouse pointer location. |
+| Middle Click Paste | Clicking the middle mouse button pastes content from the clipboard. |
+| (Touchpad) Disable While Typing | The touchpad is disabled when using the keyboard. |
+| (Touchpad) Mouse Click Emulation | Select touchpad input that will be interpreted as input from a standard mouse: <ul><li>Fingers: Click with two fingers on the touch pad to input right-click, and three fingers input a middle-click.</li><li>Area: Click the bottom right of the touch pad to input right-click, and the bottom middle for a middle-click.</li><li>Disabled: The touch pad input will not be interpreted as right, middle, or left-click actions. |
+
+### Startup Applications
+
+Choose applications that will automatically start after you log in.
+
+![Startup App Tweaks](/images/gnome-tweaks-extensions/startup-app-tweaks.png)
+
+Add an application by clicking the `+`, selecting an application from the list, and then clicking `Add`.
+
+![Select Startup App](/images/gnome-tweaks-extensions/select-startup-app.png)
+
+### Top Bar
+
+`Top Bar` settings allow you to configure the clock to display additional information.
+
+![Top Bar Tweaks](/images/gnome-tweaks-extensions/top-bar-tweaks.png)
+
+You can also choose to display week numbers in the system calendar application.
+
+![Calendar Tweaks](/images/gnome-tweaks-extensions/calendar-tweaks.png)
+
+### Window Titlebars
+
+These settings allow you to interact with windows using mouse clicks. You can also choose available titlebar buttons and their placement. Right, middle, and left-click buttons can each be configured to perform one of the following actions when clicking a window's titlebar:
+
+![Titlebar Tweaks](/images/gnome-tweaks-extensions/titlebar-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Lower | A "send to back" action occurs. |
```

Comment:
"action occurs" is a really passive way to say this. Maybe something like `The window is sent to the back.` or `The window is sent back behind any other open windows.` would work better.